MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -232,14 +234,13 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
      
      - Parameters:
         - appGroup: The App Group identifier shared by the application  with a target `Broadcast Upload Extension`
-        - videoParameters: The parameters used for limiting the screen capture resolution and target framerate
+        - videoParameters: The parameters used for limiting the screen capture resolution and target framerate, bitrate and simulcast config
         - metadata: The metadata that will be sent to the `Membrane RTC Engine` for media negotiation
-        - simulcastConfig: Simulcast configuration used for screencast track
         - onStart: The callback that will receive the screencast track called once the track becomes available
         - onStop: The callback that will be called once the track becomes unavailable
      */
-    public func createScreencastTrack(appGroup: String, videoParameters: VideoParameters, metadata: Metadata, simulcastConfig: SimulcastConfig = SimulcastConfig(), onStart: @escaping (_ track: LocalScreenBroadcastTrack) -> Void, onStop: @escaping () -> Void) -> LocalScreenBroadcastTrack {
-        let screensharingTrack = LocalScreenBroadcastTrack(appGroup: appGroup, videoParameters: videoParameters, simulcastConfig: simulcastConfig, connectionManager: connectionManager)
+    public func createScreencastTrack(appGroup: String, videoParameters: VideoParameters, metadata: Metadata, simulcastConfig: SimulcastConfig = SimulcastConfig(), maxBandwidth: TrackBandwidthLimit = .BandwidthLimit(0), onStart: @escaping (_ track: LocalScreenBroadcastTrack) -> Void, onStop: @escaping () -> Void) -> LocalScreenBroadcastTrack {
```

Comment:
shouldn't we remove `simulcastConfig` from function definition as it is in `videoParamters`